core/drivers/crypto/stm32/stm32_cryp.c

```diff
@@ -1286,6 +1327,9 @@ static TEE_Result stm32_cryp_probe(const void *fdt, int node,
 		}
 	}
 
+	if (IS_ENABLED(CFG_PM))
```

Comment:
`CFG_PM` does not exist. Can remove it.
I see that there are 2 such occurrences in stm32 drivers. We should remove them but that outside the scope of your P-R.